### PR TITLE
Add bandwidth to menu actions

### LINKED ISSUE #95: Add bandwidth toggle in long press shortcuts
Hi guys!
What an incredible work when looking through all the issues and requests!
I tried to program change VFO key (the red Exit/A/B) to another key and tried the # key (for changing from channel to freq mode) to program with a numerous key as well. If I have looked well enough I see that these keys cannot be programmed. Is that right? Could these two keys be added to the programmable keys or would that be not possible or too complicated?
Thanks so much anyway!

### PR DESCRIPTION
**Pull request by jcalado**
Fixes #95

### CODE REVIEW

File: task/keyaction.c [hunk 1]
```diff
@@ -427,6 +427,11 @@ void KeypressAction(uint8_t Action) {
 				MENU_DrawSetting();
 				break;
 
+			case ACTION_BANDWIDTH:
+				gMenuIndex = MENU_BAND_WIDTH;
+				DISPLAY_Fill(0, 159, 1, 81, COLOR_BACKGROUND);
+				MENU_DrawSetting();
```
Comment (reppad) [inline task/keyaction.c]: Hi @jcalado,
Thanks for your proposition :fire:

The idea would be to switch directly between Narrow and Wide without opening the menu.
(I haven't tested my code suggestion, you might also have to redraw the W/N indicator to update display).
```suggestion
				gVfoState[gSettings.CurrentVfo].bIsNarrow ^= 1;
```

Comment (jcalado) [reply task/keyaction.c]: my bad, I misinterpreted the issue. I'll have a look into it

Comment (jcalado) [reply task/keyaction.c]: Switched it over to a toggle and updated the display. Thanks for the tip!

File: task/keyaction.c [hunk 2]
```diff
@@ -427,6 +427,10 @@ void KeypressAction(uint8_t Action) {
 				MENU_DrawSetting();
 				break;
 
+			case ACTION_BANDWIDTH:
+				gVfoState[gSettings.CurrentVfo].bIsNarrow ^= 1;
+				UI_DrawVfo(gSettings.CurrentVfo);
```
Comment (reppad) [inline task/keyaction.c:432]: We could also save the channel to make the bandwidth change persistent.
```suggestion
				UI_DrawVfo(gSettings.CurrentVfo);
				CHANNELS_SaveVfo();
```

Comment (M7OCM) [reply task/keyaction.c:432]: Hi chaps not sure if you want to go down this route but I've added a new dialog in my fw version (plus for FM but thats not quite the same).

ui/dialog.c
	case DIALOG_BAND_WIDTH:
		if (bSet) {
			UI_DrawString(10, 48, "BW 12.5 kHz ", 12);
		} else {
			UI_DrawString(10, 48, "BW 25.0 kHz ", 12);
		}
		break;

ui/dialog.h
DIALOG_BAND_WIDTH = 3,

task/keyaction.c
UI_DrawDialogText(DIALOG_BAND_WIDTH, gVfoState[gSettings.CurrentVfo].bIsNarrow);



^^^ mine differs as I've changed quite a bit, ya'll know where to put them 👍

**Comment by reppad** [comment]
Merged, many thanks !

**Pull request merged**